internal/version/version.go

```diff
@@ -13,7 +13,6 @@ package version
 import (
 	"fmt"
 	"runtime"
-	"strings"
```

Comment:
This needs to be reverted back, it's being used.

Reply:
@jsrice7391 did you want to finish off your changes here? or would you rather me adding them? Either way is great!

Reply:
Unfortunately no. I have a lot of other things going on, thanks for letting me hack at it! Awesome tool.

Reply:
For sure, no worries, thanks!